src/Artesian/GMEPublicOffers/_Enum/Market.py

```diff
@@ -2,19 +2,23 @@
 
 
 class Market(Enum):
-    MB = 1
-    MB2 = 2
-    MB3 = 3
-    MB4 = 4
-    MB5 = 5
-    MB6 = 6
-    MGP = 7
-    MI1 = 8
-    MI2 = 9
-    MI3 = 10
-    MI4 = 11
-    MI5 = 12
-    MI6 = 13
-    MI7 = 14
-    MSD = 15
-    MRR = 16
+    MGP = 0
```

Comment:
Please do not alter fully the enumeration of the markets, just add the new markets where needed, like MIA1,2,3 can be added at the bottom of the enumeration

Reply:
> Please do not alter fully the enumeration of the markets, just add the new markets where needed, like MIA1,2,3 can be added at the bottom of the enumeration

@AndreaCuneo enumeration altered in resolution of above comment